### DISCUSSION TITLE
v1.10.0

### DISCUSSION THREAD

**Discussion by fabriciomurta** [retypeapp/retype#161] [General]
# Version 1.10.0

There were 13 changes in Retype 1.10.0.

## New

- New [KaTeX](https://retype.com/components/math-formulas/) math typesetting library support. See #73, #139.
- Upgrades [Octicons](https://retype.com/components/icon/) from v15.0.1 to v15.2.0.
- Auto generate RSS feed for blog.
- New [`server.watch.polling`](https://retype.com/configuration/project/#watchpolling) configuration option. See #151.
- [Gravatar](https://retype.com/configuration/page/#author) support for avatar. See #141.
- [Gravatar](https://retype.com/configuration/project/#gravatar): ability to disable and choose default image.
- Support [`target`](https://retype.com/components/reference-link/#target) and [`icon`](https://retype.com/components/reference-link/#custom-icon) attributes in [`[!ref]`](https://retype.com/components/reference-link/) components. See #158.
- Cleaner numerical ordering of the sidebar items.

## Fixed

- In-memory output can be logged with [`retype build`](https://retype.com/guides/cli/#retype-build).
- Blog summary page cards not updated in [`retype watch`](https://retype.com/guides/cli/#retype-watch).
- Fix support for 32-bit NodeJS installs. See #146.
- Revise component label parsing.

<hr /><em>This discussion was created from the release <a href='https://github.com/retypeapp/retype/releases/tag/v1.10.0'>v1.10.0</a>.</em>

**Comment by geoffreymcgill** [comment]
I'm loving the new Gravatar support.
